client/src/components/Navbar/Menu.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useAuth } from '../../context/useAuthContext';
+import { AuthenticatedDesktop, AuthenticatedMobile, Unauthenticated } from '../Navbar/Menus';
+
+const Menu = (): JSX.Element | null => {
+  const { loggedInUser } = useAuth();
+  const [width, setWidth] = React.useState<number>(window.innerWidth);
+  const [mobile, setMobile] = React.useState<boolean>(width < 1025);
+
+  React.useEffect(() => {
+    const handleResize = () => {
+      setWidth(window.innerWidth);
+      setMobile(window.innerWidth < 1025 ? true : false);
+    };
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, [window]);
+
+  if (!loggedInUser) return <Unauthenticated />;
+
+  if (mobile) return <AuthenticatedMobile />;
```

Comment:
since the if else statement will always resolve, i suggest writing it like

```
return ({mobile ? <AuthenticatedMobile /> : <AuthenticatedDesktop />})
```

instead of the line `return null`